airbyte-cdk/bulk/toolkits/load-low-code/src/main/kotlin/io/airbyte/cdk/load/interpolation/StringInterpolator.kt

```diff
@@ -0,0 +1,20 @@
+package io.airbyte.cdk.load.interpolation
+
+import io.pebbletemplates.pebble.PebbleEngine
+import io.pebbletemplates.pebble.loader.StringLoader
+import io.pebbletemplates.pebble.template.PebbleTemplate
+import java.io.StringWriter
+import java.io.Writer
+
+
+class StringInterpolator {
+    private val engine: PebbleEngine = PebbleEngine.Builder().loader(StringLoader()).build()
```

Comment:
I don't know all the risk in terms of malicious attacks for this templating lib so I assume there would be some validation to do here